package.json

```diff
@@ -6,10 +6,9 @@
   "scripts": {
     "server-dev": "nodemon server/server.js",
     "react-dev": "webpack -d --watch",
-    "db:setup": "node database-mongodb/seed.js",
-    "seed-db": "node database1/seed.js",
-    "generate-categories": "node database1/dataGenerator.js",
```

Comment:
I assume these wont be needed by the next devloper?

Reply:
these are the FEC commands